### PR TITLE
Admend frameles buttons in initial and disabled state, amend border c…

### PR DESCRIPTION
**Pull request by muzafarova**
…olour in night-dark

Closes https://github.com/universalbasket/css/issues/26

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -684,14 +685,22 @@ mark {
 }
 
 .button[disabled] {
-    color: var(--ui-color--mono);
-    fill: var(--ui-color--mono);
+    color: var(--ui-color--muted);
+    fill: var(--ui-color--muted);
+    background-color: var(--ui-color--default);
+    box-shadow: none;
```
Comment (kyungeunni) [inline index.css]: image

Base pure HTML button's disabled style has a shadow, but `.button` does not have a shadow, the fact that normal button's styles are identical, it looks inconsistent. but I might be wrong.

Comment (muzafarova) [reply index.css]: you're right, this should be consistent. Nice catch!

Comment (muzafarova) [reply index.css]: image

I couldn't reproduce, probably you're looking at cached version, checked that box-shadow is applied to both raw tag and a class

Comment (muzafarova) [reply index.css]: sorry, it's me who is lost. I was looking into another branch. Will fix shortly

**Pull request merged**